scrapi/base/__init__.py

```diff
@@ -262,4 +257,10 @@ def normalize(self, raw_doc):
             'dateUpdated': self.get_date_updated(result)
         }
 
+        if (result.xpath('ns0:header/@status', namespaces=self.NAMESPACES) or [''])[0] == 'deleted':
+            logger.info('Deleted record, not normalizing {}'.format(payload['id']['serviceID']))
+            return None
+        import json
+        with open('results/{}_normalized_results.json'.format(payload['source']), 'a') as f:
```

Comment:
Why are you writing this to a file here? If you want it to be for logging/record keeping purposes, it should probably be using a function defined in the core scrapi, so that it can be consistent with all of the other information we keep track of.